server/src/main/scala/com/cloudera/livy/utils/LivySparkUtils.scala

```diff
@@ -27,6 +27,22 @@ import com.cloudera.livy.util.LineBufferedProcess
 
 object LivySparkUtils {
 
+  // For each Spark version we supported, we need to add this mapping relation in case Scala
+  // version cannot be detected from "spark-submit --version".
+  private val defaultSparkScalaVersion = Map(
+    // Spark 2.0 + Scala 2.11
+    (2, 0) -> "2.11",
```

Comment:
Agree. I think we should remove the max version check. Livy can print a warning instead.